recipes/pyqtads/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyqtads" %}
+{% set version = "2.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/archive/{{ version }}.tar.gz
+  sha256: b0c83c4ced7ad25d0a090e9fa854ff7cc957523004c72046b60ab8a9d12874f5
+  patches:
+    - 0001-setup.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pyqt >=5.9
+    - qt 5.12.*
```

Comment:
are these really build deps?

Reply:
Yes. This library compiles extensions that depends on Qt and Sip...

Reply:
and pyqt is used also at the setup.py to query the Qt configuration and create resource files.